src/main/java/edu/greenblitz/tobyDetermined/commands/Auto/PathFollowerBuilder.java

```diff
@@ -0,0 +1,137 @@
+package edu.greenblitz.tobyDetermined.commands.Auto;
+
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.auto.SwerveAutoBuilder;
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.commands.swerve.SwerveCommand;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.greenblitz.utils.PIDObject;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+
+import java.sql.Time;
+import java.util.HashMap;
+
+public class PathFollowerBuilder extends SwerveAutoBuilder {
+	
+	
+	
+	public static final PIDObject translationPID = new PIDObject().withKp(0);
+	public static final PIDObject rotationPID = new PIDObject().withKp(2).withKi(0);
+	
+	/**
+	 * the use of the event map is to run a marker by its name run a command
+	 */
+	private static final HashMap<String, Command> eventMap = new HashMap<>();
+	
+	//todo add commands to event map
+	static {
+		// the event name, the command()
+		eventMap.put("place",new InstantCommand(){
```

Comment:
Do we use here more than one an Instant Command and the only difference is the string for the event map?

Reply:
When you do refactor the command to be in its own more generic command, it should also be in its own file.